CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 3.14.0 (2020-03-30)
+
+- Added the `enablePointerEvents` option to `SkyOverlayConfig` which enables click events to pass through the overlay's backdrop. [#148](https://github.com/blackbaud/skyux-core/pull/148)
+- Added the `backdropClick` event to the `SkyOverlayInstance` to allow overlay instances to listen for backdrop click events. [#148](https://github.com/blackbaud/skyux-core/pull/148)
```

Comment:
I'd either delete `the` before `SkyOverlayInstance`. (Alternately, we could add a noun after `SkyOverlayInstance`, but I'm not sure what that noun would be. Is this a component?)

Reply:
Removed "the" throughout. The `SkyOverlayInstance` is a simple class, not a component.